### PR TITLE
made collaborative vs multi-user consistent, moved multi-user to end …

### PR DESCRIPTION
**Pull request by bcarrier**
…of docs, add prereq sections to some network setup, added other references to multi-user in docs

### CODE REVIEW

File: docs/doxygen-user/installation.dox [hunk 1]
```diff
@@ -5,68 +5,28 @@
 \section prereqs Prerequisites
 It is _highly_ recommended to remove or disable any antivirus software from computers that will be processing or reviewing cases. Antivirus software will often conflict with forensic software, and may quarantine or even delete some of your results before you get a chance to look at them.
 <br><br>
+
+
+\section install Deployment Types
+Starting with Autopsy 4.0.0, there are two ways to deploy Autopsy:
```
Comment (karlmortensen) [inline docs/doxygen-user/installation.dox]: 4.0 or 4.0.0? So far, I've made the path be 4.0. Shall I change them to be 4.0.0?

File: docs/doxygen-user/installMultiUser.dox [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/*! \page install_multiuser_page Setting Up Multi-user Environment
+
+\section multiuser_install Multi-user Installation
+
+Autopsy can be setup to work in an environment where multiple users on different computers can have the same case open at the same time.  To setup this type of environment, you will need to configure additional (free and open source) network-based services.
```
Comment (karlmortensen) [inline docs/doxygen-user/installMultiUser.dox]: "To setup this type of" .... or "To set up this type of"?

File: docs/doxygen-user/installMultiUser.dox [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+We recommend using at least 2 dedicated computers for this additional infrastructure.  Spreading the services out across several machines can improve throughput. 
+If possible, place Solr on a machine by itself, as it is the largest RAM and CPU utilizer among the servers. 
+
+Ensure that the central storage and PostgreSQL servers are regularly backed up. 
+
+\subsection multiuser_install_clients Autopsy Clients
+
+Once the infrastructure is in place, you will need to configure Autopsy to use them. 
+- Install Autopsy on each client system as normal using the steps from \ref installation_page.
+- Start Autopsy and open the multi-user options panel from Tools -> Options -> Multi-user.  As shown in the screenshot below, you can then enter all of the address and authentication information for the network-based services. Note that in order to create or open Multi-user cases, "Enable Multi-user cases" must be checked and the settings below must be correct.
```
Comment (karlmortensen) [inline docs/doxygen-user/installMultiUser.dox]: I think we're using the format "Tools", Options, "Multi-user" pretty much everywhere now.

File: docs/doxygen-user/installActiveMQ.dox [hunk 4]
```diff
@@ -1,61 +1,80 @@
...
-7.	Install ActiveMQ as a service by navigating to the folder <i>apache-activemq-5.11.1-bin\\apache-activemq-5.11.1\\bin\\win64</i>, right-clicking _InstallService.bat_, clicking _Run as administrator_, then click _Yes_.
-8.  If you desire authentication for your ActiveMQ server (a good idea), the following directions allow you to set up credentials: 
-    + Copy and paste the following text to the file <i>"C:\Program Files\apache-activemq-5.11.1-bin\apache-activemq-5.11.1\conf\groups.properties"</i>, overwriting the text highlighted in yellow in the screenshot below:
-            
+
+If you do not see a screen like the above screenshot and you have double checked that the ActiveMQ service is running, contact your network administrator. For the ActiveMQ service to be accessible by network clients you may need to configure your Windows firewall (and any other 3rd party firewall in use) to allow communication.
+
+
+\subsection install_activemq_install_pw Configuring Authentication
```
Comment (karlmortensen) [inline docs/doxygen-user/installActiveMQ.dox]: From here down, the generated bullet numbers get messed up. Needs to be generated with Doxygen, and looked at.

File: docs/doxygen-user/installActiveMQ.dox [hunk 5]
```diff
@@ -1,61 +1,80 @@
...
-3.	Download ActiveMQ-5.11.1 from: http://activemq.apache.org/activemq-5111-release.html
-4.	Extract the files in the archive
-5.	Edit <i>apache-activemq-5.11.1\\conf\\activemq.xml</i> to add <i>"&amp;wireFormat.maxInactivityDuration=0"</i> to the URI for the _transportConnector_ named _openwire_. Add the text highlighted in yellow below:
+
+
+3.	Install ActiveMQ as a service by navigating to the folder <i>bin\\win64</i>, right-clicking _InstallService.bat_, clicking _Run as administrator_, then click _Yes_.
+
+4.  Start the ActiveMQ service by pressing _Start_, type _services.msc_, and press _Enter_. Find _ActiveMQ_ in the list and press the _Start the service_ link.
+
+5. ActiveMQ should now be fully installed and configured. You can access the admin pages in your web browser via a URL like this (set your host): http://localhost:8161/admin. The default administrator username is _admin_ with a password of _admin_ and the default regular username is _user_ with a default password of _password_.  You can change these passwords by following the instructions below. If you can see a page that looks like the following, it is ready to function.
```
Comment (karlmortensen) [inline docs/doxygen-user/installActiveMQ.dox]: I don't call it fully configured by this point. They really should add some authentication setup. Or we should at least say that it's installed, but optional configuration can be made.

Comment (bcarrier) [inline docs/doxygen-user/installActiveMQ.dox]: Changed.

File: docs/doxygen-user/installActiveMQ.dox (hunk 4 above)
Comment (bcarrier) [inline docs/doxygen-user/installActiveMQ.dox]: Fixed, I think.

File: docs/doxygen-user/installMultiUser.dox (hunk 2 above)
Comment (bcarrier) [inline docs/doxygen-user/installMultiUser.dox]: Changed to "set up"

File: docs/doxygen-user/installMultiUser.dox (hunk 3 above)
Comment (bcarrier) [inline docs/doxygen-user/installMultiUser.dox]: Fixed.

File: docs/doxygen-user/installation.dox (hunk 1 above)
Comment (bcarrier) [inline docs/doxygen-user/installation.dox]: Changed to 4.0

**Pull request merged**